UCDlist.tex

```diff
@@ -607,10 +618,13 @@ \section{List of valid words}
 Q & \ucd{src.var.pulse}& Pulse\\
 Q & \ucd{stat}& Statistical parameters\\
 Q & \ucd{stat.asymmetry}& Measure of asymmetry\\
+P & \ucd{stat.confidenceLevel}& Level of confidence for a detection or classification process\\
 P & \ucd{stat.correlation}& Correlation between two parameters\\
 P & \ucd{stat.covariance}& Covariance between two parameters\\
 P & \ucd{stat.error}& Statistical error\\
 P & \ucd{stat.error.sys}& Systematic error\\
+Q & \ucd{stat.falsePositive}& Rate of false alarm in detection problems\\
+Q & \ucd{stat.falseNegative}& Rate of missed or false negative detection\\
```

Comment:
Is there a rationale for making them Q?  Where would these be in a secondary position?